src/PowerShellEditorServices/Services/PowerShell/Host/EditorServicesConsolePSHost.cs

```diff
@@ -1,362 +1,50 @@
-﻿using Microsoft.Extensions.Logging;
```

Comment:
The new public host is just a wrapper that allows us to make our host public without exposing anything dangerous to users